python/lsst/dax/images/cutout/_image_cutout.py

```diff
@@ -21,7 +21,7 @@
 
 from __future__ import annotations
 
-__all__ = ("ImageCutoutBackend", "Extraction")
+__all__ = ("Extraction", "ImageCutoutBackend")
```

Comment:
Open question is whether `ImageCutoutBackend` is still the right name now that it's `lsst.dax.images.cutout` and not `lsst.image_cutout_backend`.  Options?

* `ImageCutoutFactory`
* `ImageCutoutMaker`
? Or versions without `Image` prefix..